db/external_sst_file_ingestion_job.cc

```diff
@@ -792,18 +792,20 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     }
     file_to_ingest->smallest_internal_key.SetFrom(key);
 
-    iter->SeekToLast();
-    auto largest = iter->key();
+    Slice largest;
     if (strcmp(cfd_->ioptions()->table_factory->Name(), "PlainTable") == 0) {
       // Not Support SeekToLast , have to iter all key
       // mostly would happen in PlainTable
       iter->SeekToFirst();
```

Comment:
Need to check iter->Valid() before calling iter->key() here too. Can also just skip SeekToFirst() since it's already called above and Valid() is checked.

Reply:
I deleted SeekToFirst, but there is no  iter->key() in these lines.

Reply:
You mean line 800?

Reply:
Fixed it.